catalog/data_metastores.go

```diff
@@ -0,0 +1,20 @@
+package catalog
+
+import (
+	"context"
+
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func DataSourceMetastores() *schema.Resource {
+	return common.AccountData(func(ctx context.Context, data *MetastoresData, acc *databricks.AccountClient) error {
```

Comment:
This is added in https://github.com/databricks/terraform-provider-databricks/pull/2429/files